src/client/components/FloatButton/FloatButton.component.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Please just add a button with the text Back that should go to the Landing page. Let's keep things simple and not add packages with icons

Reply:
1. there is no packages ...
2. its just an image where it looks nice and also how can user press "back" when he comes from any page to category or product .. "back" does not mean anything here .. Thats why i have mentioned comment(which shows go to home) when you hover it ...

Reply:
I need you to respect the design, this button does not respect the design https://github.com/HackYourFuture-CPH/fp-class18/blob/floatingBtn/src/client/assets/images/homeicon.png 

Please remove the images from this PR and add a button with text saying "Back".